clients/geth/specular/data/.gitignore

```diff
@@ -1,4 +1,5 @@
 data
 data_sequencer
 data_validator
-data_indexer
\ No newline at end of file
+data_indexer
```

Comment:
think we can get rid of all the `data_*` from here since they're under `data/` now